homework11/task1_metaclasses/tests/test_hw11_t1.py

```diff
@@ -0,0 +1,16 @@
+from project_hw11_task1.task1 import SimplifiedEnum
+
+
+class ColorsEnum(metaclass=SimplifiedEnum):
+    __keys = ("RED", "BLUE", "ORANGE", "BLACK")
+
+
+class SizesEnum(metaclass=SimplifiedEnum):
+    __keys = ("XL", "L", "M", "S", "XS")
+
+
+def test_attributes_are_accessible():
```

Comment:
You test two different classes in a single test.  By the way what the difference between these classes?

Reply:
in names of classes, class attributes

Reply:
What is special in these names and attributes? 

Let's split this test into 2 and name each one according to its special feature.